test/decl/var/property_wrappers.swift

```diff
@@ -849,13 +849,30 @@ struct WrapperWithProjectedValue<T> {
   var projectedValue: T { return wrappedValue }
 }
 
-class TestInvalidRedeclaration {
+class TestInvalidRedeclaration1 {
+  // expected-error@-1 {{invalid redeclaration of synthesized property '_i'}}
+  // expected-error@-2 {{invalid redeclaration of synthesized property '$i'}}
```

Comment:
Similarly here, I worry that these diagnostics do more harm than good. It is enough that the user is warned about duplicate wrapper redeclarations; there is nothing actionable by pointing these out, but now there are three errors emitted when there is only one error. I think the lack of an explicit location on which to pin these diagnostics is a good hint that they're not actionable.

Reply:
@theblixguy I think you can attach this note to the original wrapped property decl via `VarDecl::getOriginalWrappedProperty`

Reply:
That's a really good idea! I have done that now.